Make SplunkPy parse multiple values of same key to list (#36130)
demisto/content: Packs/SplunkPy/ReleaseNotes/3_1_38.md

#### Integrations

##### SplunkPy

> Fixed an issue where **splunk-parse-raw** command overrode same key values instead of listing all of them.

Reviewer: ```suggestion
Fixed an issue where the **splunk-parse-raw** command ignored values in case their key appeared more than once.
```
